getting-started/gopher-run/cmd/main.go

```diff
@@ -89,6 +139,40 @@ func (a *app) topScores(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (a *app) addPlayData(w http.ResponseWriter, r *http.Request) {
+	bkt := a.bucket
+	var d playData
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&d); err != nil {
+		fmt.Fprintf(w, "decoder.Decode: %v", err)
+	}
+	r.Body.Close()
+	new := bkt.Object("pldata/" + d.Name + "_pldata.csv").NewWriter(r.Context())
+	defer new.Close()
+	new.Write([]byte(d.Value))
+	fmt.Fprint(w, "Recieved data\n")
+}
+
+// SendGeneratedBackground returns cloud/hill placements.
+func (a *app) SendGeneratedBackground(w http.ResponseWriter, r *http.Request) {
+	var d generator.RequestData
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&d); err != nil {
+		fmt.Fprintf(w, "decoder.Decode: %v\n", err)
+		log.Printf("decoder.Decode: %v\n", err)
+		return
+	}
+	r.Body.Close()
+	generator.Speed = d.Speed
```

Comment:
Avoid global variables when possible. Can this be an argument to `GenerateBackground` instead?